satellite/api.go

```diff
@@ -108,8 +109,8 @@ type API struct {
 	}
 
 	Payments struct {
-		Accounts payments.Accounts
-		Clearing payments.Clearing
+		Accounts  payments.Accounts
+		Inspector *stripecoinpayments.Endpoint
```

Comment:
It's really not about inspecting, it's about other things as well.

Reply:
I called it like this so it is easier to understand where we should invoke this